.github/workflows/simorgh-amp-a11y-validation.yml

```diff
@@ -24,13 +24,20 @@ jobs:
         uses: actions/setup-node@v1
```

Comment:
**nitpick:** is it worth renaming this file now given that we're grouping a number of tests which run when the server is running

Reply:
renaming it now